src/cmd_all/src/playground.rs

```diff
@@ -143,6 +143,16 @@ pub struct PlaygroundOpts {
     profile: String,
 }
 
+impl risingwave_common::opts::Opts for PlaygroundOpts {
+    fn name() -> &'static str {
+        "playground"
+    }
+
+    fn meta_addr(&self) -> &str {
+        "0.0.0.0:5690" // hard-coded
+    }
```

Comment:
can we do it in the yaml file? hard code is always a bad practice.

Reply:
I believe the meta address for playground is intentionally hard-coded. 😕

https://github.com/risingwavelabs/risingwave/blob/1922b0169a8cf55857a56421f7ef5b52beb008dc/src/cmd_all/src/playground.rs#L59